### ISSUE TITLE
Alma login form fields not associated with their labels

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/orangelight#3216]
### Expected behavior
The Alma user name and Password fields on the login page are associated with their labels.  This way, a screen reader will appropriately announce the field labels when a user is in the field.

### Actual behavior
The <label> "for" attributes do not match the <input> ids:
- `<label [...] for="user_Alma user name">Alma user name</label>`
- `<input id="username" [...]`
- `<label [...] for="user_Password">Password</label>`
- `<input [...] id="password" [...]`

VoiceOver simply announces the fields as "edit text" and "secure edit text"

### Steps to replicate

1. Go to https://catalog-staging.princeton.edu/users/sign_in
2. Press the "Log In With Alma Account" button
3. Turn on voiceover
4. Tab into the username and password field
5. Confirm that voiceover simply says "edit text" and "secure edit text", rather than the actual names of the field.

### Impact of this bug

Screen reader users wouldn't be able to log in using alma accounts without some serious guess work.

### Miscellaneous thoughts

I'm pretty sure that `for="user_Alma user name"` is auto-generated by simple_forms.  It's not valid HTML, since the [for attribute can only point to a single ID](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for), and [IDs cannot contain whitespace](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id).

### Further reading

- [WebAIM's guide on accessible forms](https://webaim.org/techniques/forms/controls)

**Comment by christinach** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/OrangelightBibdataRequests-571691cab409d8d821b873be/issues/pulibrary/orangelight/3216?planning-poker) with Zenhub @sandbergja

**Comment by christinach** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/OrangelightBibdataRequests-571691cab409d8d821b873be/issues/pulibrary/orangelight/3216?planning-poker) with Zenhub @maxkadel @leefaisonr
